web/src/message_scroll.js

```diff
@@ -150,13 +150,18 @@ export function initialize() {
             return;
         }
 
-        if (event.mark_read && event.previously_selected_id !== -1) {
-            // Mark messages between old pointer and new pointer as read
```

Comment:
I changed the comments here to refer to the selected message and not pointer as that doesn't seem to be a term we use in the code base much anymore.

Reply:
Yeah, the pointer used to be a thing in the API -- and was important before we tracked unread messages individually, but was a bad model and has been mostly purged.